Installation/CMakeLists.txt

```diff
@@ -8,27 +8,16 @@ if(NOT PROJECT_NAME)
   project(CGAL CXX C)
 endif()
 
-
-if(POLICY CMP0056)
-  # https://cmake.org/cmake/help/v3.2/policy/CMP0056.html
-  #
-  # Fix a bug: `try_compile` should use `CMAKE_EXE_LINKER_FLAGS`.
-  # That variable can contain important flags like
-  #
-  #     -static-libstdc++ -static-libgcc
-  #
-  cmake_policy(SET CMP0056 NEW)
-endif()
-
-# Use GNUInstallDirs to get canonical paths
-include(GNUInstallDirs)
-
-#--------------------------------------------------------------------------------------------------
+# https://cmake.org/cmake/help/v3.2/policy/CMP0056.html
 #
-#                                    -= HEADER ONLY =-
+# Fix a bug: `try_compile` should use `CMAKE_EXE_LINKER_FLAGS`.
+# That variable can contain important flags like
 #
-#--------------------------------------------------------------------------------------------------
-option(CGAL_HEADER_ONLY "Enable header-only mode of CGAL (no compilation of CGAL libraries)" TRUE)
+#     -static-libstdc++ -static-libgcc
+#
+cmake_policy(SET CMP0056 NEW)
```

Comment:
I think for now one, we should require CMake>=3.12, and use `cmake_minimum_required(VERSION 3.12...3.18)` everywhere.

That will remove all policies before `CMP0076`.